### DISCUSSION TITLE
matrix room for typst

### DISCUSSION THREAD

**Discussion by basings** [typst/typst#3032] [General]
I recently read about typst on lemmy (bte there's not yet a typst community) and usually my first step for such a project is to enter the matrix room.

Just to say hi and read and chat about the project. Any newcomer can enter the chatroom and isn't left alone.

I'd appreciate if you'd consider opening a room for people to chat and hangout. What's your take on this?

**Comment by Enivex** [comment]
There's an active discord

**Reply by basings** [reply]
What are you implying?

**Reply by Enivex** [reply]
That you can join the Discord server if you want chat?

**Reply by basings** [reply]
I asked because the last issue was confusing.

I do not intend to join discord which is why I was asking about matrix. Matrix is much more developer friendly.

**Reply by Enivex** [reply]
It's come up before, and there are no plans to use Matrix

**Reply by basings** [reply]
That's very unfortunate

**Reply by basings** [reply]
Anyway, I just found the typst room on matrix via matrix.to

**Comment by basings** [comment]
If anyone is looking for a matrix room, there is matrix.to/#/#typst:matrix.org
